randomizer/Enums/Enemies.jsonc

```diff
@@ -101,6 +101,7 @@
     "KRoolChunky": 109,
     "Bug": 110,
     "FairyQueen": 111,
-    "IceTomato": 112
+    "IceTomato": 112,
+    "Beanstalk": 113
```

Comment:
See buildenums comment

Reply:
On second thought, I can see you're using this elsewhere. This enum is meant to tie to how enemy spawners work, so whilst adding an extra entry makes sense for the calculation purposes, this really disassociates it with it's purpose.

Reply:
Which is to say that I'm conflicted